src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -274,27 +285,149 @@ export class QuickBar extends LitElement {
+            domainToName(this.hass.localize, domain)
+          )
+      ),
       icon: domainIcon(domain),
       action: () => this.hass.callService(domain, "reload"),
     }));
   }
 
+  private _generateServerControlCommandItems(): QuickBarItem[] {
+    const serverActions = ["restart", "stop"];
+
+    return serverActions.map((action) =>
+      this._generateConfirmationCommandItem(
+        {
+          text: this.hass.localize(
+            "ui.dialogs.quick-bar.commands.server_control.perform_action",
+            "action",
+            this.hass.localize(
+              `ui.dialogs.quick-bar.commands.server_control.${action}`
+            )
+          ),
+          icon: "hass:robot",
+          action: () => this.hass.callService("homeassistant", action),
+        },
+        this.hass.localize("ui.dialogs.generic.default_confirmation_title")
+      )
+    );
+  }
+
+  private _generateConfirmationCommandItem(
+    item: QuickBarItem,
+    confirmText: ConfirmationDialogParams["confirmText"]
+  ): QuickBarItem {
+    return {
+      ...item,
+      action: () =>
+        showConfirmationDialog(this, {
+          confirmText,
+          confirm: item.action,
+        }),
+    };
+  }
+
+  private getNavigationInfo(sectionKey: string, component: string) {
+    const panel = configSections[sectionKey].find(
+      (section) => section.component === component
+    );
+
+    if (!panel || !panel.translationKey) {
+      throw Error("Navigation section or its translation not found");
+    }
+
+    const text = this.hass.localize(
+      "ui.dialogs.quick-bar.commands.navigation.navigate_to",
+      "panel",
+      this.hass.localize(
+        `ui.dialogs.quick-bar.commands.navigation.${component}`
+      )
+    );
+
+    return { ...panel, text };
+  }
+
+  private _getPanelText(panel: Panels["panel"]) {
+    let translationKey = "";
+
+    if (!panel.title) {
+      switch (panel.component_name) {
+        case "profile":
+          translationKey = "panel.profile";
+          break;
+        case "lovelace":
+          translationKey = "panel.states";
+          break;
+      }
+    } else {
+      translationKey = `panel.${panel.title}`;
+    }
+
+    return this.hass.localize(
+      "ui.dialogs.quick-bar.commands.navigation.navigate_to",
+      "panel",
+      toTitleCase(this.hass.localize(translationKey))
+    );
+  }
+
+  private _generateNavigationPanelItems(): Omit<
+    QuickBarNavigationItem,
+    "action"
+  >[] {
+    return Object.keys(this.hass.panels).map((panelKey) => {
+      const panel = this.hass.panels[panelKey];
+
+      return {
+        text: this._getPanelText(panel),
+        icon: "hass:robot",
+        path: `/${panel.url_path}`,
+      };
+    });
+  }
+
+  private _generateNavigationSectionItems(): Partial<QuickBarNavigationItem>[] {
+    return [
+      this.getNavigationInfo("general", "logs"),
+      this.getNavigationInfo("automation", "automation"),
+      this.getNavigationInfo("automation", "script"),
```

Comment:
You should check if these integrations are loaded, check <https://github.com/home-assistant/frontend/blob/dev/src/panels/config/dashboard/ha-config-navigation.ts#L30-L33>

And why not just all the items in `configSections` + cloud? <https://github.com/home-assistant/frontend/blob/dev/src/panels/config/dashboard/ha-config-dashboard.ts>

Reply:
> You should check if these integrations are loaded, 

Ahh, right. I remember thinking about that early on and then totally forgot 😆 

> And why not just all the items in configSections + cloud?

Thought I should start slow :) See what the demand/use-cases were. But let's go for the gold!

Reply:
Oh... and I guess I should also check for whether "Advanced Mode" is checked on certain items?

Reply:
I think the code is easier if we do it all ;-)

Reply:
Yes and no (but mostly yes).

The issue is that we can't automatically lazy load the strings needed from configSections (yet... I have a branch I'm working on that might give us this some day). 

So every new page added to `configSections` will need to manually add a string entry to `en.json` that points to the appropriate caption in `ui::panels::config`.

Reply:
Pushed support for all pages. Still need to do advanced mode checks

Reply:
Added integration existence check and advanced mode check. Refactored into helper method to be reused by config page